packages/types/lib/scripts/http.api.ts

```diff
@@ -6,3 +6,19 @@ export type GetPublicScriptsConfig = Endpoint<{
     Path: `/scripts/config`;
     Success: StandardNangoConfig[];
 }>;
+
+export interface OpenAIFunction {
+    name: string;
+    description: string;
+    parameters: {
+        type: string;
+        properties: Record<string, any>;
+        required: string[];
+    };
+}
+
+export type GetPublicScriptsConfigOpenAI = Endpoint<{
+    Method: 'GET';
+    Path: `/scripts/config/openai`;
+    Success: OpenAIFunction[];
```

Comment:
Can you make the export `{ data: OpenAIFunction[] }` is easier for consistency and if we ever implement pagination

Reply:
Yeah I thought about pagination too. Agreed. 

Also realized missing the open spec endpoint documentation so it shows in our docs reference